src/Form/OgMenuInstanceForm.php

```diff
@@ -296,22 +326,82 @@ protected function buildOverviewTreeForm($tree, $delta) {
    * {@inheritdoc}
    */
   public function save(array $form, FormStateInterface $form_state) {
-    $entity = $this->entity;
-    $status = parent::save($form, $form_state);
+    $menu = $this->entity;
+    if (!$menu->isNew() || $menu->isLocked()) {
```

Comment:
Is the menu an entity of an OgMenuInstance? Because the method isLocked() in OgMenuInstance is empty.